README.md

```diff
@@ -46,6 +46,15 @@ Disclaimer: GitHub Gists are by default **public**. If you don't want other peop
     personalAccessToken: "6a10cc207b....7a67e871"
 ```
 
+### Cloning a backup to a fresh Atom install
+
+1. Install the package from the command line: `apm install sync-settings`
+1. Launch Atom passing in **GITHUB_TOKEN** and **GIST_ID**. For example:
+
+```
+GITHUB_TOKEN=6a10cc207b....7a67e871 GIST_ID=b3025...88c41c atom
```

Comment:
These instructions are certainly helpful. But afterwards the user still has to setup the GitHub token and Gist ID in the package settings. Can you please update this paragraph to explicitly mention that the above instructions are still necessary for that.